manifests/examples/kvm/jessie-lvm.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: debian-lvm-example
+provider:
+  name: kvm
+bootstrapper:
+  workspace: /target
+system:
+  release: jessie
+  architecture: amd64
+  bootloader: grub
+  charmap: UTF-8
+  locale: en_US
+  timezone: UTC
+volume:
+  backing: lv
+  logicalvolume:
```

Comment:
I don't think there's any need to put `vg` and `lv` under another key. Just spell them out directly in `volume`. So: `volume: {logivalvolume: "lvname", volumegroup: "vgname"}`